shell/android-studio/reicast/src/main/java/com/reicast/emulator/config/OptionsFragment.java

```diff
@@ -767,4 +764,34 @@ private void showToastMessage(String message, int duration) {
 				.getDimensionPixelOffset(R.dimen.snackbar_icon_padding));
 		snackbar.show();
 	}
+
+	private int getBroadcastValue(String broadcastName) {
+		if (broadcastName.equals("NTSC-J"))
```

Comment:
Isn't that something that should be done inside reicast core to avoi duplicate code?

Reply:
It's only used in two places and under review. If the values are incorrect, the replacement code does not require any sort of helper. It was done this way to make for a simple removal.